otter/test/test_auth.py

```diff
@@ -77,10 +77,45 @@ def test_extract_token(self):
         resp = {'access': {'token': {'id': u'11111-111111-1111111-1111111'}}}
         self.assertEqual(extract_token(resp), '11111-111111-1111111-1111111')
 
-    def test_authenticate_user(self):
+    def test_authenticate_user_with_pool(self):
```

Comment:
Possibly these two tests can be split into "Validate that authenticate_user works and processes treq responses correctly", and "Validate that the pool argument is passed to treq", in which case this particular test probably does not need to replicate everything `test_authenticate_user_without_pool` is doing.

Alternately, the actual testing of the treq response processing can be factored out into a helper function rather than copy-pasted?

Not going to block on this, just an observation for the future, etc.  :)